### ISSUE TITLE
Feature Request: more functions in history section

### ISSUE THREAD

**Issue by droople** [ReDEnergy/SessionSync#62]
Hi,

I like SessionSync, and use it everyday.

Personally, I don't like to save my session to bookmark and sync across all my firefox browsers, so I mainly use the history section.

I'm just wondering that if possible that you add more functions into history section?

Such as
1) always only keep 10 histories no matter how many histories saved;
2) automatically save a history can be set in seconds,minutes, days, weeks, etc 

Regards

**Comment by ReDEnergy** [comment]
Yeah... the history section really needs a serious complete update because it lacks a lot of features that should be considered standard. Since I never used auto-saving myself and based on a few request I decided to implement it but at the same time I made some strange decisions on how it works.
I already started working on an improved version that is suppose to offer much more feature and at the same time be more reliable.

Write now the only options you are able to set are these:
![image](https://user-images.githubusercontent.com/1395510/52402999-59d0da80-2ace-11e9-8048-637bfc6c6233.png)

What exactly do you expect from the **1.** feature. Is it something different that the **Saving entries** options that keeps only that amount of histories or do you mean something else. Yes in my case there's also the auto-delete feature but if you set that to a really great number won't delete any saves. Now that I think about it... the delete option should be optional because you might still want to have the previous 10 saves even if a lot of time has passed between them.

I'll keep both ideas in mind for the updated version because they make sense.
